src/sass/components/_index.scss

```
@@ -4,3 +4,4 @@
 @import './_mobile-menu';
 @import './_contact-phones-mobile-menu';
 @import './_contact-phones';
+@import './button';
```

Надо заменить на правильный путь)) Делай по аналогии
```suggestion
@import './_button';
```